src/Sulu/Bundle/SecurityBundle/Tests/Functional/Controller/ResettingControllerTest.php

```diff
@@ -197,7 +199,8 @@ public function testResendEmailAction()
         $this->assertEquals($this->user3->getEmail(), $response->email);
 
         // asserting user properties
-        $user = $this->em->find('SuluSecurityBundle:User', $this->user3->getId());
+        $user = $client->getContainer()->get('doctrine')->getManager()
+            ->find('SuluSecurityBundle:User', $this->user3->getId());
```

Comment:
It is using the container from the client application, not the application created via. `$this->getContainer`. In this case it probably wouldn't matter as the `user` is *probably* persisted to the database, but I am not sure if it is actually required. Does it matter?